src/AWS.Deploy.Orchestration/DeploymentBundleHandler.cs

```diff
@@ -135,12 +135,20 @@ private void SwitchToSelfContainedBuildIfNeeded(Recommendation recommendation)
                 if (string.IsNullOrEmpty(targetFramework))
                     return;
 
-                // Elastic Beanstalk doesn't currently have .NET 7 and 9 preinstalled.
-                var unavailableFramework = new List<string> { "net7.0", "net9.0" };
-                var frameworkNames = new Dictionary<string, string> { { "net7.0", ".NET 7" }, { "net9.0", ".NET 9" } };
-                if (unavailableFramework.Contains(targetFramework))
+                // Elastic Beanstalk currently has .NET 8 and 9 supported on their platforms.
+                var supportedFrameworks = new List<string> { "net8.0", "net9.0" };
```

Comment:
is there any way to query beanstalk to see what their supported versions are instead of updating this list every time?

Reply:
Not aware of anywhere besides their documentation. We can't rely on the platform name since they also have .NET Core which supports different versions.